website/docs/cli/commands/completion.mdx

```diff
@@ -28,6 +30,72 @@ When the generated completion script is loaded into the shell, pressing the tab
 Run `atmos completion --help` to see all the available options
 :::
 
+
+
+<Tabs>
+  <TabItem value="bash" label="Bash" default>
+## Bash Completion Setup
+
+To enable tab completion for Atmos in `bash`, follow the steps given bellow
+
+```
+# execute and capture the autocompletion script
+atmos completion bash &> ~/.atmos_completion.sh
+
+# add the permissions
+chmod +x ~/.atmos_completion.sh
+```
+
+Add the following line to your `.bashrc`
+```
+source ~/.atmos_completion.sh
+```
```

Comment:
Please use instructions from PR comments

Reply:
ohh this should not be here

Reply:
Done